modules/i18n/src/translations/en-US/portals/remote-user-stores.ts

```diff
@@ -0,0 +1,245 @@
+                        },
+                        heading: "Custom Attributes"
+                    },
+                    heading: "Attribute Mappings",
+                    local: {
+                        heading: "Local Attributes"
+                    }
+                },
+                groupAttributes: {
+                    heading: "Group attributes"
+                },
+                userAttributes: {
+                    heading: "User attributes"
+                }
+            },
+            guide: {
+                heading: "Connect the remote user store",
+                steps: {
+                    attributeMapping: {
+                        description:
+                            "Update the <1>attribute mappings</1> according to remote user store that you connected. Make sure to review the mapped attributes otherwise it may cause errors in the user listings.",
+                        heading: "Attribute mappings"
+                    },
+                    configure: {
+                        description:
+                            "Update the properties in the deployment.toml file located in the root directory of the user store agent to match the remote user store settings. Add additional properties according to your requirements.",
+                        docsDescription:
+                            "See the <1>Asgardeo documentation</1> for more details on configuring the user store agent.",
+                        heading: "Configure the agent"
+                    },
+                    download: {
+                        heading: "Download the agent",
+                        onPrem: {
+                            action: "Download the agent",
+                            description: "Download and unzip the user store agent."
+                        },
+                        remote: {
+                            description: "Select your platform and download the corresponding user store agent:"
+                        }
+                    },
+                    run: {
+                        checkConnection: {
+                            action: "Check Connections",
+                            errorAction: "Try again",
+                            errorHeading: "Not connected",
+                            errorMessage:
+                                "A user store is not connected, please make sure that you have followed all the steps of the setup guide properly.",
+                            successAction: "Successfully Connected"
+                        },
+                        commands: {
+                            unix: "On Linux/Mac OS",
+                            windows: "On Windows"
+                        },
+                        description: {
+                            onPrem:
+                                "Execute one of the following commands based on your operating system. Enter the installation token on prompt.",
```

Comment:
```suggestion
                                "Execute one of the following commands based on your operating system. Enter the installation token when prompted.",
```

Reply:
Already available content will be kept as it is. The component has been separated into a file.